src/cpp/rtps/xmlparser/XMLParser.cpp

```diff
@@ -468,7 +468,7 @@ XMLP_ret XMLParser::parseXMLCommonTransportData(
                     p_aux1 != nullptr; p_aux1 = p_aux1->NextSiblingElement())
             {
                 address = p_aux1->Name();
-                if (strcmp(address, ADDRESS) == 0)
+                if (strcmp(address, ADDRESS) == 0 || strcmp(address, INTERFACE_NAME) == 0)
```

Comment:
I wouldn't have added a new tag, is there any reason for this? I mean, they are treated identically after all.

Reply:
Update comment at beginning of function.